app/assets/stylesheets/alchemy/jquery.datetimepicker.scss

```diff
@@ -0,0 +1,545 @@
+  margin-top: 8px;
+  margin-bottom: 3px
+}
+
+.xdsoft_datetimepicker .xdsoft_mounthpicker {
+  position: relative;
+  text-align: center;
+}
+
+.xdsoft_datetimepicker .xdsoft_label i,
+.xdsoft_datetimepicker .xdsoft_prev,
+.xdsoft_datetimepicker .xdsoft_next,
+.xdsoft_datetimepicker .xdsoft_today_button {
+  background-image: none;
+}
+
+.xdsoft_datetimepicker .xdsoft_label i {
+  opacity: 0.5;
+  display: inline-block;
+  width: 9px;
+  height: 20px;
+  vertical-align: middle;
+
+  &:before {
+    content: '';
+    position: absolute;
+    top: 50%;
+    right: 0;
+    transform: translateY(-50%);
+    width: 0;
+    height: 0;
+    border-style: solid;
+    border-width: 4px 4px 0 4px;
+    border-color: $text-color transparent transparent transparent;
+  }
+}
+
+.xdsoft_datetimepicker .xdsoft_prev,
+.xdsoft_datetimepicker .xdsoft_next {
+  position: relative;
+
+  &:before {
+    content: '';
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    width: 0;
+    height: 0;
+    border-style: solid;
+  }
+}
+
+.xdsoft_datetimepicker .xdsoft_prev {
+  float: left;
+
+  &:before {
+    border-width: 7px 7px 7px 0;
+    border-color: transparent $text-color transparent transparent;
+  }
+}
+
+.xdsoft_datetimepicker .xdsoft_today_button {
+  float: left;
+  margin: 0 5px;
+
+  &:before {
+    font-family: 'Alchemy Icons';
+    font-size: 16px;
+    line-height: 18px;
+    content: '\e00a';
+  }
+}
+
+.xdsoft_datetimepicker .xdsoft_next {
+  float: right;
+
+  &:before {
+    border-width: 7px 0 7px 7px;
+    border-color: transparent transparent transparent $text-color;
+  }
+}
+
+.xdsoft_datetimepicker .xdsoft_next,
+.xdsoft_datetimepicker .xdsoft_prev ,
+.xdsoft_datetimepicker .xdsoft_today_button {
+  background-color: transparent;
+  background-repeat: no-repeat;
+  border: 0 none;
+  cursor: pointer;
+  display: block;
+  height: 30px;
+  opacity: 0.5;
+  -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=50)";
+  outline: medium none;
+  overflow: hidden;
+  padding: 0;
+  position: relative;
+  white-space: nowrap;
+  width: 20px;
+  min-width: 0;
+}
+
+.xdsoft_datetimepicker .xdsoft_timepicker .xdsoft_prev,
+.xdsoft_datetimepicker .xdsoft_timepicker .xdsoft_next {
+  float: none;
+  height: 15px;
+  width: 30px;
+  display: block;
+  margin-left: 14px;
+  margin-top: 7px;
+}
+
+.xdsoft_datetimepicker.xdsoft_rtl .xdsoft_timepicker .xdsoft_prev,
+.xdsoft_datetimepicker.xdsoft_rtl .xdsoft_timepicker .xdsoft_next {
+  float: none;
+  margin-left: 0;
+  margin-right: 14px;
+}
+
+.xdsoft_datetimepicker .xdsoft_timepicker .xdsoft_prev {
+  margin-bottom: 8px;
+  margin-top: 0;
+
+  &:before {
+    border-width: 0 8px 8px 8px;
+    border-color: transparent transparent $text-color transparent;
+  }
+}
+
+.xdsoft_datetimepicker .xdsoft_timepicker .xdsoft_next {
+
+  &:before {
+    border-width: 8px 8px 0 8px;
+    border-color: $text-color transparent transparent transparent;
```

Comment:
Shorthand form for property `border-color` should be written more concisely as `$text-color transparent transparent` instead of `$text-color transparent transparent transparent`